custom/icds_reports/reports/new_born_with_low_weight.py

```diff
@@ -32,14 +33,17 @@ def get_data_for(filters):
             queryset = apply_exclude(domain, queryset)
         return queryset
 
+    month = datetime(*config['month'])
```

Comment:
It looks like this operation is also done in `get_data_for` is there a way to have it only done once per function. Maybe modifying it here in the main body and removing that line from `get_data_for`

Reply:
done